cmd/registry/cmd/auth/print_token.go

```diff
@@ -24,7 +24,7 @@ import (
 func printTokenCommand() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "print-token",
-		Short: "Invoke token source and print token.",
+		Short: "Invoke token-source to print a token.",
```

Comment:
What happens if no token source is set? Would/should it default to printing a manually-set registry.token?

Reply:
(Is `registry.token` going away? I notice it's not displayed by `registry config list` I'm not sure that it's actually getting set when I run `registry config set registry.token ... ` the `get` command isn't returning a value)

Reply:
It's not possible to manually set one in a configuration currently. The user would simply use the --registry.token flag.